pennylane/templates/layers/strongly_entangling.py

```diff
@@ -21,19 +21,19 @@
 from pennylane.templates.utils import (
     check_shape,
     check_no_variable,
-    check_wires,
     check_type,
     check_number_of_layers,
     get_shape,
 )
+from pennylane.wires import Wires
 
 
 def strongly_entangling_layer(weights, wires, r, imprimitive):
     r"""A layer applying rotations on each qubit followed by cascades of 2-qubit entangling gates.
 
     Args:
         weights (array[float]): array of weights of shape ``(len(wires), 3)``
-        wires (Sequence[int]): sequence of qubit indices that the template acts on
+        wires (Wires): sequence of qubit indices that the template acts on.
```

Comment:
Do you think users directly use this function much? Since this would be a breaking change for them, i.e., now only supporting `Wires` input :thinking:

Reply:
No, not at all. It should be made private in fact! Shall I refactor to underscore?

Reply:
I think it's ok, it's not visible in the docs anyway :grin: